File: Python/1486.py
# 1486. XOR Operation in an Array
# https://leetcode.com/problems/xor-operation-in-an-array/

class Solution:
  def xorOperation(self, n: int, start: int) -> int:
    out = 0

    for i in range(0, n):
      out ^= start
      start += 2

    return out

File: Rust/1486.rs
/**
 * 1486. XOR Operation in an Array
 * https://leetcode.com/problems/xor-operation-in-an-array/
 */

impl Solution {
  pub fn xor_operation(n: i32, start: i32) -> i32 {
    let mut start: i32 = start;
    let mut out: i32 = 0;

    for i in 0..n {
      out ^= start;
      start += 2;
    }

    return out;
  }
}
